lib/main.js

```diff
@@ -0,0 +1,16 @@
+"use 6to5";
+let NVAtom = require("./NVAtom");
+
+exports.activate = (state) => {
+  atom.nvatom = new NVAtom(state);
```

Comment:
If you did `this.nvAtom = new NVAtom(state)` then you could access it elsewhere via `atom.packages.getActivePackage('nvatom').mainModule.nvAtom`.

Another option would be to use services, https://github.com/atom/atom/pull/5277